nginx/prod.nginx.conf

```diff
@@ -1,11 +1,24 @@
 limit_req_zone $binary_remote_addr zone=api_limit:10m rate=25r/s;
-limit_req_zone $binary_remote_addr zone=nginx_limit:10m rate=30r/s;
+limit_req_zone $binary_remote_addr zone=nginx_limit:10m rate=40r/s;
 
 upstream api {
   least_conn;
   server api:8081 weight=10 max_fails=3 fail_timeout=30s;
 }
 
+proxy_cache_path /nginx-cache levels=1:2 keys_zone=my_cache:30m inactive=60m use_temp_path=off max_size=3g;
```

Comment:
Since we don't have this enabled for local environment, have you tested the cache? I'm wondering specifically if it can distinguish different query params for the same path.

Reply:
After deploying this, I can see that the `random-components` does not get cached since new components are received with every request. Also, the `/nginx-cache` directory is empty.

Reply:
Can you please elaborate on that? I'm not sure how new components are being received.

Reply:
It looks like the `random-component` calls are not cached by NGINX, since new data is received every time, meaning that the request is passed via the API to Neo4j.

Reply:
Alright, the caching problems have been fixed - all requests are cached except for `random-components`.